src/base58.cpp

```diff
@@ -224,32 +224,39 @@ class CBitcoinAddressVisitor : public boost::static_visitor<bool>
 {
 private:
     CBitcoinAddress* addr;
+    bool IsColdStakingAddr = false;
 
```

Comment:
Instead of have a boolean here, you could have the address prefix directly. It will be easier for new address types to be added and the code will look cleaner without the bool param everywhere.

Reply:
Absolutely. Great idea :+1:

Reply:
In https://github.com/PIVX-Project/PIVX/pull/955/commits/b1bb33b86a18819bb1a44c62feb031e08dc3b6f0